app/helpers/shift_commerce/bundle_helper.rb

```diff
@@ -0,0 +1,10 @@
+# freeze_string_literal: true
+module ShiftCommerce
+  module BundleHelper
```

Comment:
Can these be called something that links them to being a cache helper??
```suggestion
  module BundleCacheHelper
```